### PR TITLE
adding logic whether to print star or all columns in a query

### PR DESCRIPTION
**Pull request by saurabhgupta-datametica**

### CODE REVIEW

File: core/src/main/java/org/apache/calcite/config/QueryStyle.java [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.apache.calcite.config;
+
+/**
+ *this class is used to set some config property from mig side.
+ */
+public class QueryStyle {
+  public boolean isExpandStar() {
```
Comment (yogeshmuley) [inline core/src/main/java/org/apache/calcite/config/QueryStyle.java:23]: can it be expandProjection?

Comment (saurabhgupta-datametica) [reply core/src/main/java/org/apache/calcite/config/QueryStyle.java]: changes the variable name to expandProjection

**Review by yogeshmuley — COMMENTED** [review]
please add a test case in ReltoSqlConverterTest.

**Comment by saurabhgupta-datametica** [comment]
> please add a test case in ReltoSqlConverterTest.

when we create rel in reltoSqlConverter the rel created for this type of test case does not have project above table scan , so it is not possible to write test case for this type of scenario

**Review by yogeshmuley — APPROVED** [review]

**Pull request merged**